src/endpoints/metrics.js

```diff
@@ -117,6 +117,15 @@ metricsRouter.get("/editedCount/:resourceType", (req, res, next) => {
     .catch(next)
 })
 
+metricsRouter.get("/templateUsageCount", (req, res, next) => {
+  console.log(req.query.templateId)
+  req.db
+    .collection("resources")
+    .count({ templateId: req.query.templateId })
+    .then((response) => res.send({ count: response }))
+    .catch(next)
+})
```

Comment:
I am not totally confident in this query.  Do we identify the templates used to create a resource by querying for the `templateId`?  Based on inspecting some documents, I think this is correct, but would be good to verify.

Reply:
Verified this query should be correct